tasks/winbuildscripts/lint.ps1

```diff
@@ -8,7 +8,7 @@ if ($Env:TARGET_ARCH -eq "x64") {
 & $Env:Python3_ROOT_DIR\python.exe -m  pip install -r requirements.txt
 
 $LINT_ROOT=(Get-Location).Path
-$Env:PATH="$LINT_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python3_ROOT_DIR;$Env:Python3_ROOT_DIR\Scripts;$Env:PATH"
+$Env:PATH="$LINT_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python3_ROOT_DIR;$Env:Python3_ROOT_DIR\Scripts;$Env:PATH;C:\devtools\vstudio\VC\Tools\Llvm\bin"
```

Comment:
the root of vstudio is defined in an env variable; so if for some reason it moves this code wouldn't need to be changed.  IIRC it's `VSTUDIO_ROOT`